wger/core/templates/forms/password_with_toggle.html

```diff
@@ -0,0 +1,92 @@
+<!-- templates/forms/password_with_toggle.html -->
+<div class="password-wrapper" style="position: relative;">
+    <input type="{{ widget.type }}" name="{{ widget.name }}"{% if widget.value %} value="{{ widget.value }}"{% endif %}{% if widget.attrs.id %} id="{{ widget.attrs.id }}"{% endif %}{% if widget.attrs.class %} class="{{ widget.attrs.class }}"{% endif %}{% if widget.attrs.placeholder %} placeholder="{{ widget.attrs.placeholder }}"{% endif %}{% if widget.attrs.required %} required{% endif %}{% if widget.attrs.maxlength %} maxlength="{{ widget.attrs.maxlength }}"{% endif %}{% if widget.attrs.autocomplete %} autocomplete="{{ widget.attrs.autocomplete }}"{% endif %}>
+    <button type="button" class="password-toggle-btn" aria-label="Toggle password visibility" style="position: absolute; right: 10px; top: 50%; transform: translateY(-50%); background: none; border: none; cursor: pointer; padding: 5px; z-index: 10;">
+        <svg class="eye-icon eye-open" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+            <path d="M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z"></path>
+            <circle cx="12" cy="12" r="3"></circle>
+        </svg>
+        <svg class="eye-icon eye-closed" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" style="display: none;">
```

Comment:
you can use the font awesome icons here, they also have ones with an open and a crossed out eye